Demo/src/main/java/com/braintreepayments/demo/PayPalFragment.java

```diff
@@ -55,9 +60,7 @@ public void launchBillingAgreement(View v) {
     }
 
     private void launchPayPal(boolean isBillingAgreement) {
-        FragmentActivity activity = getActivity();
-        activity.setProgressBarIndeterminateVisibility(true);
-
+        FragmentActivity activity = requireActivity();
```

Comment:
Same here can we pass `requireActivity()` every time instead?

Reply:
Reduced the number of locally scoped activity references. Adding a next major version node to refactor `PayPalClient` to allow tokenization without the need for an activity reference.